### ISSUE TITLE
Export 2d plan view elements creates an invalid geometric representation context

### ISSUE THREAD

**Issue by Moult** [Autodesk/revit-ifc#187]
If the option is enabled, the following syntax is created:

```
#108= IFCGEOMETRICREPRESENTATIONCONTEXT($,'Annotation',3,0.01,#96,#97);
#109= IFCGEOMETRICREPRESENTATIONSUBCONTEXT($,'Annotation',*,*,*,*,#108,0.01,.PLAN_VIEW.,$);
```

This is invalid. From the docs:

 > NOTE  The inherited attribute ContextType shall have one of the following recognized values: 'Model', 'Plan', 'NotDefined'.

**Comment by AngelVelezSosa** [comment]
That's correct.  Exporting 2d plan view elements is strictly speaking not supported in CV2.0 - it was in CV.  That option allows you to do "extra", which is why it isn't enabled by default.

**Comment by Moult** [comment]
@AngelVelezSosa I'm not talking about whether it that option is enabled or not. What I'm saying is that when it is enabled, it produced invalid IFC syntax. The docs require "Model" or "Plan" for ContextType, but the produced IFC does not follow that. What should happen is that Annotation is created as a subcontext under a Plan context.

**Comment by AngelVelezSosa** [comment]
I was trying to look up the docs for CV and CV2.0- where are you looking?

**Comment by Moult** [comment]
For IFC2X3

https://standards.buildingsmart.org/IFC/RELEASE/IFC2x3/TC1/HTML/ifcrepresentationresource/lexical/ifcgeometricrepresentationcontext.htm

For IFC4 and onwards

https://standards.buildingsmart.org/IFC/RELEASE/IFC4/FINAL/HTML/link/ifcgeometricrepresentationcontext.htm

Both have the same restriction of Model or Plan or NotDefined.

**Comment by Moult** [comment]
@AngelVelezSosa can you confirm this issue?

**Comment by Moult** [comment]
Bump.

**Comment by AngelVelezSosa** [comment]
Yup, easy fix.  Will do.  "Plan" context, "Annotation" subcontext.

**Comment by ArtemHnatko** [comment]
The issue is not reproducible in Revit 2023-2025. The exported IFC files now correctly adhere to the IFC schema requirements:
- The ContextType is set to "Plan" for the main context.
- The "Annotation" is correctly created as a subcontext under the "Plan" context.

**Testing environment:**
- Autodesk Revit 2023 (v.23.1.60.0)
- Autodesk Revit 2024
- Autodesk Revit 2025

![Image](https://github.com/user-attachments/assets/c424b943-9687-436f-a216-697326c1c3df)
